src/webapi/events/pointer.rs

```diff
@@ -0,0 +1,483 @@
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-tangentialpressure
+    #[inline]
+    fn tangential_pressure( &self ) -> f64 {
+        js!(
+            return @{self.as_ref()}.tangentialPressure;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the angle, in the range of [-90, 90] degrees, between the Y-Z plane and the plane
+    /// containing the transducer (e.g. pen stylus) and the Y axis.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tiltX)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-tiltx
+    #[inline]
+    fn tilt_x( &self ) -> i32 {
+        js!(
+            return @{self.as_ref()}.tiltX;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the angle, in the range of [-90, 90] degrees, between the X-Z plane and the plane
+    /// containing the transducer (e.g. pen stylus) and the X axis.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tiltY)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-tilty
+    #[inline]
+    fn tilt_y( &self ) -> i32 {
+        js!(
+            return @{self.as_ref()}.tiltY;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the clockwise rotation, in the range of [0, 359] degrees, of
+    /// the transducer (e.g. pen stylus) around it's own major axis
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/twist)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-twist
+    #[inline]
+    fn twist( &self ) -> i32 {
+        js!(
+            return @{self.as_ref()}.twist;
+        ).try_into().unwrap()
+    }
+
+    /// Indicates the device type that caused the event.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pointerType)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-pointertype
+    #[inline]
+    fn pointer_type( &self ) -> String {
+        js!(
+            return @{self.as_ref()}.pointerType;
+        ).try_into().unwrap()
+    }
+
+    /// Indicates if the pointer represents the primary pointer of this pointer type
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/isPrimary)
+    // https://w3c.github.io/pointerevents/#dom-pointerevent-isprimary
+    #[inline]
+    fn is_primary( &self ) -> bool {
+        js!(
+            return @{self.as_ref()}.isPrimary;
+        ).try_into().unwrap()
+    }
+
+    /// Returns the sequence of all `PointerEvent` instances that were coalesced into the dispatched `PointerMoveEvent`.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/getCoalescedEvents)
+    // https://w3c.github.io/pointerevents/extension.html#dom-pointerevent-getcoalescedevents
+    #[inline]
+    fn get_coalesced_events( &self ) -> Vec<PointerEvent> {
```

Comment:
> Returns the sequence of all `PointerEvent` instances that were coalesced into the dispatched `PointerMoveEvent`.

Shouldn't this be defined in `PointerMoveEvent` then? Even the specs says that it `always returns at least one coalesced event for pointermove events and an empty list for other types of PointerEvents.`